Formula/darktable.rb

```diff
@@ -0,0 +1,82 @@
+class Darktable < Formula
+  desc "Open source photography workflow application and raw developer"
```

Comment:
@dunn there is no "RAW" image format.
This is a direct quote from our official site:
https://www.darktable.org/
```
darktable is an open source photography workflow application and raw developer.
```

But yes, that is what meant here.